src/term/mod.rs

```diff
@@ -1031,6 +1112,32 @@ impl ansi::Handler for Term {
     }
 
     #[inline]
+    fn set_cursor_style(&mut self, mode: ansi::CursorType) {
+        trace!("set_cursor_style: {:?}", mode);
+
+        match mode {
+            ansi::CursorType::BlinkBlock => {
```

Comment:
Not understanding what's happening here. Blinking needs to be handled separately by an event loop rather than expecting escape codes to show up periodically.